prog/filter_test.go

```diff
@@ -0,0 +1,35 @@
+package prog
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestFilterArguments(t *testing.T) {
+	tests := [][2]string{
+		{
+			`dfetch0(&(0x7f0000000000)='123')`,
+			"false",
+		},
+		{
+			`dfetch1(&(0x7f0000000000)={0x0, {0x0}})`,
+			"true",
```

Comment:
Wait, but isn't this test supposed to return false, as we discussed yesterday?

Reply:
yeah yeah it is returning false now, i am just making some modification to the code to handle this case and i will push it with the right expected return value